### PR TITLE
Feature/terraform applyrunandcreatevariable

### PR DESCRIPTION
**Pull request by KalyankarRajesh**
Created the sl files for both terraform create variable and apply run.

Created and executed the test cases and everything working fine.

### CODE REVIEW

File: content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+namespace: io.cloudslang.hashicorp.terraform.runs
+
+operation: 
+  name: apply_run
+  
+  inputs:
+    - auth_token:
+        sensitive: true
+        required: true
```
Comment (IuliaMoldovan) [inline content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl]: remove required:true as true is the default value

File: content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    - authToken:
+        default: ${get('auth_token', '')}
+        required: true
```
Comment (IuliaMoldovan) [inline content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl]: also here

File: content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl (hunk 1 above)
Comment (KalyankarRajesh) [reply content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl]: incorporated review comments.

File: content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl (hunk 2 above)
Comment (KalyankarRajesh) [reply content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl]: incorporated review comments.

File: content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+  
+  inputs:
+    - auth_token:
+        sensitive: true
+    - authToken:
+        default: ${get('auth_token', '')}
+        private: true
+        sensitive: true
+    - run_id:
+        required: true
```
Comment (florinAndrei1991) [inline content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl]: this can be removed as it's redundant ... by default the value of required is true

Comment (KalyankarRajesh) [reply content/io/cloudslang/hashicorp/terraform/runs/apply_run.sl]: incorporated the review comments

**Pull request merged**